src/services/collections/vacancies.ts

```diff
@@ -75,45 +75,88 @@ export const vacancies: any[] =   [
       'GraphQL knowledge is a plus;',
       'UX principles;',
       'HTML5, responsiveness;',
       'Continuous integration experience;',
-      'Familiarity and understanding of Agile Methodologies;',
-      'Experience in building modern web-apps with cross-browser support (i.e NativeScript, Ionic, Electron);',
       'Intermediate+ written and oral English;',
       'Experience with VCS (git, github).'
     ],
-    headerImage: 'assets/images/vacancy/vacancy.jpg'
+    headerImage: 'assets/images/vacancy/vacancy.jpg',
+    location: 'Kharkiv, Ukraine'
+  },
+  {
+    vacancyId: 3,
+    name: 'Project Manager',
+    description: 'We are seeking for an ambitious self-starter with a positive attitude. The right Project Manager must have excellent communication skills, written as well as verbal as you would be liaising with both internal and external stakeholders and maintain close contact with team members. Technical proficiency, and ability to handle any technical queries effectively for Project Manager is a must. This role would be a big step on your way to becoming a Senior Project Manager position within a truly pioneering IT company. ',
+    responsibilities: [
+      'Typical activities within this project support role include: generation and maintenance of cross functional project plans, organization, and documentation of project meetings, creation / analysis / reporting of project metrics, coordination with stakeholders;',
+      'Leads critical project governance activities, including facilitating project meetings, progressing projects through life cycle transitions, and ensuring quality standards are maintained through quality assurance activities;',
+      'Develops and implements risk assessment, change management, quality assurance, performance, and control measures to ensures completion of projects by established standards;',
+      'Early identification of issues and/or concerns that either has the potential or will impact the project baseline;',
+      'Providing accurate and timely information, data and reports regarding the project status, activities and performance.'
+    ],
+    offer: [
+      'Passionate & fun team to work with;',
+      'Competitive salary;',
+      'Fast, professional growth and lots and lots of new experience are guaranteed;',
+      'Partial grant of the cost of certification and related conferences;',
+      'Friendly and relaxed atmosphere;',
+      'Flexible working hours;',
+      'Extremely good coffee.'
+    ],
+    requirements: [
+      'At least one year of experience in managing technology/online/business projects in a professional environment involving multiple stakeholders;',
+      'At least one year experience in managing teams;',
+      'Knowledge of IT operation processes such as risk management;',
+      'Strong estimation skills;',
+      'Knowledge of Agile project management approaches;',
+      'Excellent time management, organizational skills, and communication skills;',
```

Comment:
organizational skills, and communication skills
maybe organizational and communication skills ?